src/main/java/core/basesyntax/BalanceOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
According to the checklist, there should not be an empty line at the beginning of a class or method implementation. Please remove the empty line (line 2).